src/main/java/io/github/minecraftcursedlegacy/api/vanillachecker/VanillaChecker.java

```diff
@@ -0,0 +1,19 @@
+package io.github.minecraftcursedlegacy.api.vanillachecker;
+
+import java.util.HashMap;
+
+import io.github.minecraftcursedlegacy.impl.vanillachecker.VanillaCheckerImpl;
+import net.minecraft.entity.player.Player;
+
+public class VanillaChecker {
```

Comment:
Having the vanilla checker a package only for it seems unneccesary. Perhaps use an `api/server` package, and this could be a `ServerUtils` class.

Reply:
Also the class could be made final while you're at it but not neccesary

Reply:
moved to api/server keeping class name